roles/redpanda_broker/tasks/main.yml

```diff
@@ -1,49 +1,39 @@
 ---
-- name: Check if redpanda_version is defined
-  fail:
-    msg: "A valid value must be provided for redpanda_version ex: '22.2.2-1' or 'latest'"
-  when: redpanda_version is not defined
-
-- name: Check if ca_cert_file is defined
-  fail:
-    msg: "A valid path must be provided for ca_cert_file ex: tls/certs/ca.crt"
+- name: Ensure redpanda_version is defined
+  assert:
+    that:
+      - redpanda_version != ''
+    fail_msg: "Variable 'redpanda_version' must be defined!"
```

Comment:
we've lost context in the error message I think. What should an example redpanda version number be (or latest)?